docs/vcl.md

```diff
@@ -0,0 +1,80 @@
+## Introduction and Overview
+
+VCL is a communication library in VPP that provides a socket-based interface for client/server communication with VPP. It is designed to enable client applications to establish connections with VPP, send and receive control messages, and perform configuration and management operations.
```

Comment:
Maybe something like :

VCL stands for the [VPP Comms Library](https://wiki.fd.io/view/VPP/HostStack/VCL), it is a library that allows application to easily leverage the VPP hoststack (i.e. TCP, UDP, TLS, QUIC, ... stack implementation in VPP itself) in a similar way as the libc works for kernel hoststack syscalls. 
It works by exposing a unix socket over which control messages exchanged and configurations negotiated.